python/tskit/tables.py

```diff
@@ -585,6 +586,42 @@ def _update_metadata_schema_cache_from_ll(self) -> None:
             self.ll_table.metadata_schema
         )
 
+    def metadata_vector(self, key, *, dtype=None, default_value=None):
+        """
+        Returns a numpy array of metadata values obtained by extracting ``key``
+        from each metadata entry, and inserting ``default_value`` if the key is
```

Comment:
"inserting" suggests that we might be modifying the metadata
```suggestion
        from each metadata entry, and using ``default_value`` if the key is
```